vulnerabilities/fi/file3.php

```diff
@@ -20,10 +20,11 @@
 		[<em><a href=\"?page=include.php\">back</a></em>]
 	</div>
 
-	<h2>More info</h2>
+	<h2>More Information</h2>
 	<ul>
-		<li>" . dvwaExternalLinkUrlGet( 'https://en.wikipedia.org/wiki/Remote_File_Inclusion' ) . "</li>
-		<li>" . dvwaExternalLinkUrlGet( 'https://www.owasp.org/index.php/Top_10_2007-A3' ) . "</li>
+	<li>" . dvwaExternalLinkUrlGet( 'https://en.wikipedia.org/wiki/Remote_File_Inclusion', 'Wikipedia - File inclusion vulnerability' ) . "</li>

```

Comment:
Can you add this link to the help.php as well. May as well give all references in all places.